recipes/casadi/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "casadi" %}
+{% set version = "3.1.1" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/casadi/casadi.git
+  git_rev: {{ version }}
```

Comment:
I see some `numpy.get_include()` in the source code, so the answer is probably yes.